pandas/core/strings.py

```diff
@@ -646,9 +646,13 @@ def str_split(arr, pat=None, n=None, return_type='series'):
     """
     from pandas.core.series import Series
     from pandas.core.frame import DataFrame
+    from pandas.core.index import Index
 
     if return_type not in ('series', 'frame'):
```

Comment:
the `return_type` parameter is a bit tricky now that `.str` works for both `Index` and `Series`, for instance you can imagine someone doing this

```
s = Series(['a_b_c', 'd_e', 'f', 'g'])
s.str.split('_', return_type='index')
```
Do you think this should return a `Series` or an `Index`?

(similarly for `return_type='series'` on an `Index` str operation.)